modules/reactor/doc/content/source/meshgenerators/PolygonConcentricCircleMeshGenerator.md

```diff
@@ -75,6 +75,15 @@ By default, the mesh generated by this object contains a mixture of QUAD4 and TR
       id=TRI3_and_QUAD4
       caption=A schematic drawing showing the different meshing algorithms when [!param](/Mesh/PolygonConcentricCircleMeshGenerator/quad_center_elements) is set as `false` and `true`.
 
+Users can also set [!param](/Mesh/PolygonConcentricCircleMeshGenerator/replace_inner_ring_with_delaunay_mesh) to true to replace the inner ring with an unstructured Delaunay triangular mesh as illustrated in [#inner_delaunay]
+
+!media reactor/meshgenerators/tri_delaunay.png
+      style=display: block;margin-left:auto;margin-right:auto;width:40%;
+      id=inner_delaunay
+      caption=Example meshes when [!param](/Mesh/PolygonConcentricCircleMeshGenerator/replace_inner_ring_with_delaunay_mesh is set as `false` and `true`.
+
+This feature is useful in a situation when the number of total sectors is large making the center point connected with too many elements. When this parameter is true, [!param](/Mesh/PolygonConcentricCircleMeshGenerator/quad_center_elements) is ignored and users will get a warning if this parameter is set to true. The first element of [!param](/Mesh/PolygonConcentricCircleMeshGenerator/ring_intervals) is ignored and the element areas are controlled by [!param](/Mesh/PolygonConcentricCircleMeshGenerator/inner_ring_desired_area) function. The function can be a constant value. The default empty string means that the code will pick a default desired area which is the square of the smallest side length on the ring circle. The unstructured Delaunay mesh of the inner ring will have the triangle element type specified in [!param](/Mesh/PolygonConcentricCircleMeshGenerator/tri_element_type). It will also have the subdomain ID of the original subdomain ID of the inner ring. When [!param](/Mesh/PolygonConcentricCircleMeshGenerator/ring_id_name) is specified, the element ring IDs will be all equal to 1. When [!param](/Mesh/PolygonConcentricCircleMeshGenerator/sector_id_name) is specified, the element sector IDs will be all equal to 0.
```

Comment:
There are parameters in addition to `quad_center_elements` and `ring_intervals`, that will be ignored. Including the block id and name for the quad elements of the inner ring, as well as the mesh bias and boundary layer parameters

Reply:
Updated. Actually, the code uses the value of the first element in `ring_block_ids` to assign block ids and similarly with `ring_block_names`.

Reply:
if `ring_intervals[0]`>1, the second elements of `ring_block_ids` and `ring_block_names` are ignored. Also, if the new feature is used, the first elements of
`ring_inner_boundary_layer_biases`
`ring_inner_boundary_layer_intervals`
`ring_inner_boundary_layer_widths`
`ring_outer_boundary_layer_biases`
`ring_outer_boundary_layer_intervals`
`ring_outer_boundary_layer_widths`
`ring_radial_biases`
are all ignored.

Reply:
I think this is the only unresolved comment of mine. Please add these to the doc.

@GiudGiud Do you want to do an additional round of review?

Reply:
Oh, I thought one sentence without this list is fine. I will push this then.